Makefile

```diff
@@ -0,0 +1,62 @@
+SHELL := /bin/bash
```

Comment:
Here's an awesome addition to Makefiles: automatic help (with sections): https://github.com/flashbots/go-template/blob/main/Makefile#L1-L11